Add Administrative commands

kirbykirby56 opened the pull request:
Adds GetOther, ListOther, and DeleteOther.
Permissions imageonmap.<>other


As a side note I suck at using git and that's why this PR is 7 commits, only one of which actually changes anything lol.

Commit:
Merge pull request #1 from zDevelopers/indev

update to zdev off indev

Commit:
Add administrative commands

Commit:
Removed debug-command

Commit:
Revert to 3d7d2fd

Because I screwed up.

Commit:
Add administrative commands

Commit:
Revert to 3d7d2fd again

Because git is really hard for beginners

and i keep including ImageMap.java

Commit:
Add administrative commands

Vlammar commented on src/main/java/fr/moribus/imageonmap/commands/maptool/DeleteOtherCommand.java:
```diff
@@ -0,0 +1,98 @@
+		ImageMap map = MapManager.getMap(uuid, mapName);
+        
+        if(player != null) MapManager.clear(player.getInventory(), map);
+        //if(player == null) MapManager.clear(op.getPlayer().getInventory(), map);
```
Forgotten commented code.

Vlammar commented on src/main/java/fr/moribus/imageonmap/commands/maptool/ListOtherCommand.java:
```diff
@@ -0,0 +1,124 @@
+                .then(map.getId())
+                .color(ChatColor.WHITE)
+                .command(GetCommand.class, map.getId())
+                .hover(new ItemStackBuilder(Material.MAP)
```
This should be an issue like for #107 Should be replace by `.hover(new RawText()
                        .then(map.getName()).style(ChatColor.BOLD, ChatColor.GREEN).then("\n")
                        .then(map.getId() + ", " + size).color(ChatColor.GRAY).then("\n\n")
                        .then(I.t("{white}Click{gray} to get this map"))
                );`

Vlammar commented on src/main/resources/help/maptool.txt:
```diff
@@ -3,6 +3,7 @@ new: Creates a new ImageOnMap
 get: Gives you a map.
+getother: Gets another player's map.
 getremaining: Gives you the remaining maps that could not fit in your inventory
 list: Lists all the map you currently have.
```
Should add: `listother: list all the map of another player`

Vlammar commented on src/main/resources/help/maptool.txt:
```diff
@@ -3,6 +3,7 @@ new: Creates a new ImageOnMap
 delete: Deletes a map.
 delete-noconfirm: Deletes a map. Deletion is permanent and made without confirmation
```
Add: deleteother: Deletes another map.

Vlammar reviewed (changes requested):
Missing deleteother.txt : Deletes another player's map.

§6<playername>: §rThe name of the player who owns the map.
§6<mapname>: §rThe name of the map. Use /maptool listother to get map names.

Vlammar committed:
Update DeleteOtherCommand.java

Removed commented code and inverted permission check with arg length check. Changed success and error messages

Vlammar committed:
Update maptool.txt

Vlammar committed:
Update maptool.txt

Vlammar committed:
Update listother.txt

Vlammar committed:
Update ListOtherCommand.java

Vlammar committed:
Merge branch 'indev' into Imbadatgit
